build/config.props

```diff
@@ -33,7 +33,8 @@
     <VsTargetBranch>master</VsTargetBranch>
     <VsTargetChannel>int.$(VsTargetBranch)</VsTargetChannel>
     <VsTargetBranch Condition="'$(IsEscrowMode)' == 'true'">rel/d$(VsTargetMajorVersion).$(MinorNuGetVersion)</VsTargetBranch>
-    <VsTargetChannel Condition="'$(IsEscrowMode)' == 'true'">Preview</VsTargetChannel>
+    <!-- VS short branch name. See https://github.com/dotnet/arcade/blob/master/Documentation/ArcadeSdk.md#visual-studio-training-inputs-generation -->
```

Comment:
I'm not sure what arcade has to do with this. 
We are not onboarded to arcade

Reply:
I would avoid this link and just inline a distilled explanation.